### DISCUSSION TITLE
Change titles in checkout summary area

### DISCUSSION THREAD

**Discussion by madmaud** [woocommerce/woocommerce#51759] [Checkout Flow]
It's great that we can edit some checkout titles in the input fields but we really need to be able to match them in the summary area. For example, I need to change "Shipping" to "Delivery" but there's no way to update the summary field where "Shipping" appear to "Delivery." Thanks!

**Comment by alexflorisca** [comment]
Thanks starting the discussion, this feels like a bug we should address. I don't see any reason why we couldn't make the Order Summary fields (Coupon,Tax, Shipping) editable as well, what do you think @woocommerce/rubik?

**Reply by opr** [reply]
We could maybe use entities to store these. Worth exploring, not sure how it looks from a product point of view? @pmcpinto what do you think of making these things in the order summary editable?

**Reply by pmcpinto** [reply]
Thanks for the ping. That can be useful in some use cases. But I don't think we should allow merchants to change the shipping method name, as this is defined at the admin settings level. Makes sense?
Can someone from the team create a GH issue about this?

**Reply by opr** [reply]
> But I don't think we should allow merchants to change the shipping method name

I don't think this is the intention, just the top-level labels in the order summary, e.g. Total, Subtotal, Shipping, Tax, Fees.

I'll create the issue.

**Reply by pmcpinto** [reply]
Thank you. Feel free to cc me on the issue. We need to ensure that users easily understand which summary areas can be edited.

**Comment by nielslange** [comment]
The corresponding GitHub issue for this discussion can be found on https://github.com/woocommerce/woocommerce/issues/52043.

**Comment by ralucaStan** [comment]
I created this issue to take a look at that Delivery label https://github.com/woocommerce/woocommerce/issues/53927
@madmaud care to explain your usecase; how are you changing the title, how do you want to change the Delivery label?

**Reply by webfolkcreative** [reply]
@ralucaStan Found this when looking for a similar answer.

In my use case, if we offer "Local Pickup" as a method, the word "Collection" feels awkward. I would rather change this to "Local Pickup", with the pickup location showing below (as the address currently does). No one uses the word Collection, it's always local pickup or curb-side pickup. It would be nice to be able to modify this heading.

**Reply by nielslange** [reply]
@webfolkcreative In #54746, @helgatheviking changed the wording from "Collection" to "Pickup". You mentioned that it's always "local pickup" or "curb-side pickup", so the label "Pickup" would capture both variants.

**Reply by madmaud** [reply]
Apologies @ralucaStan, I think you already closed this- for some reason I didn't see the email notifications until today. 
I'll explain my use-case anyway in case it helps for future changes.

We are using WooCommerce for items that are for delivery only. There is literally no shipping going on but it's nigh impossible to get the checkout verbiage to reflect that. If the user can't update the field names to whatever they want, then an option for selecting physical delivery (and as mentioned above- pick up) instead of just shipping that would update the titles of the fields appropriately would be great. Just some way of getting the fields to update appropriately when a particular delivery method is selected. Right now I've had to resort to the classic layout in order to add a bunch of custom code just to change text on the screen for the cart and checkout pages.

For example, the cart page has the sidebar with Cart Totals. We need for the shipping option to say "Delivery" instead. On the checkout page, I've changed Billing and Shipping to Billing and Delivery and under Your Order, changed Billing to Delivery. I've made additional customizations based on our needs but it would be wonderful to have some more options on those text changes from the UI. 
That's really it. Again, my apologies for such a late reply.

**Reply by ralucaStan** [reply]
@madmaud no worry, thank you for coming back. It's really helpful. @yjailin and @pierorocca this discussion is relevant for making some order summary fields editable.

Also, in WooCommerce 9.8 we'll ship some improvements for the shipping/local pickup labels, among which #54746; It might be worth trying the Checkout block again to see if it better fits with your needs. You can use https://woocommerce.com/products/woocommerce-beta-tester/ or build the trunk branch.


>  I've made additional customizations based on our needs but it would be wonderful to have some more options on those text changes from the UI.

We're working on bringing further improvements to the Local pickup area. Let me know if the latest changes work for you, or if have extra feedback.

**Reply by pierorocca** [reply]
Also adding that in general Shipping sections and labels have been changed to Delivery so that we're inclusive of shipping, local pickup, and local delivery use cases. "Shipping" wasn't inclusive enough so thanks for bringing this to our attention.
